Makefile

```diff
@@ -95,6 +95,12 @@ E2E_TEST_FLAGS = "--skip-deletion=false" -timeout 25m # See README.md, default g
 # see target "image-build"
 IMAGE_BUILD_FLAGS ?= --build-arg USE_LOCAL=false
 
+# Prometheus-Unit Tests Parameters
+PROMETHEUS_CONFIG_YAML = ./config/monitoring/prometheus/apps/prometheus-configs.yaml
```

Comment:
So, you've decided not to use $(CURDIR) instead of `pwd`. Fine for me. I guess there is no difference here if you use full path or the relative path. Also `./` a bit redundant probably, but may be brings more clarity.